### PR TITLE
Inspiration credits to lit protocol

### PR DESCRIPTION
**Pull request by KPrasch**
2 approvals please!

### CODE REVIEW

File: nucypher/policy/conditions/lingo.py [hunk 1]
```diff
@@ -123,7 +123,12 @@ def eval(self, data) -> bool:
 
 
 class ConditionLingo:
-    # TODO: 'A Collection of re-encryption conditions evaluated as a compound boolean condition'
+    """
+    A Collection of re-encryption conditions evaluated as a compound boolean condition.
```
Comment (KPrasch) [inline nucypher/policy/conditions/lingo.py:127]:
```suggestion
    A Collection of re-encryption conditions evaluated as a compound boolean expression.
```

File: nucypher/policy/conditions/lingo.py [hunk 2]
```diff
@@ -123,7 +123,12 @@ def eval(self, data) -> bool:
...
+
+    This is an alternate implementation of the condition expression format used in the Lit Protocol,
+    credit to the authors for inspiring this work. https://github.com/LIT-Protocol
```
Comment (derekpierre) [inline nucypher/policy/conditions/lingo.py:130]:
```suggestion
    This is an alternate implementation of the condition expression format used in the Lit Protocol (https://github.com/LIT-Protocol);
    credit to the authors for inspiring this work. 
```

**Review by derekpierre — APPROVED** [review]
🎸 . One suggestion.

File: nucypher/policy/conditions/lingo.py (hunk 2 above)
Comment (cygnusv) [reply nucypher/policy/conditions/lingo.py:130]: 🔥

**Review by cygnusv — APPROVED** [review]

**Pull request merged**